src/v0/destinations/tiktok_ads/util.js

```diff
@@ -21,6 +21,7 @@ const getContents = (message) => {
         price: product.price,
         quantity: product.quantity,
         description: product.description,
+        brand: product.brand
```

Comment:
did you check if anything else is missing?

Reply:
Yeah this was also caught when checking for if all the parameters that tiktok supports is mapped by us